src/test/DeltaNeutralLp.t.sol

```diff
@@ -203,21 +216,83 @@ contract L1DeltaNeutralTest is TestPlus {
         // Max borrowable WETH available in AAVE in this block is around 450k WETH or 730e6 USDC (ethereum).
         // Have to restrict debt size tro $20M since that's how there's available to borrow on polygon
         // Size of biggest position is debt * 4/3 * 7/4 = debt * 7/3
-        assets = bound(assets, 0, uint256(20e6 * 1e6) * 7 / 3);
+        // not going for whole range as for weth the pool is not that large.
+        assets = bound(assets, (10 ** (asset.decimals() - 1)) + 1, uint256(100 * startAssets));
 
-        if (assets > 1e5) {
-            assertEq(strategy.totalLockedValue(), 0);
+        assertEq(strategy.totalLockedValue(), 0);
 
-            deal(address(asset), address(strategy), assets);
-            assertApproxEqRel(strategy.totalLockedValue(), assets, 0.01e18);
+        deal(address(asset), address(strategy), assets);
+        assertApproxEqRel(strategy.totalLockedValue(), assets, 0.01e18);
 
-            vm.startPrank(vault.governance());
-            strategy.startPosition(IDEAL_SLIPPAGE_BPS);
-            assertApproxEqRel(strategy.totalLockedValue(), assets, 0.01e18);
+        vm.startPrank(vault.governance());
+        strategy.startPosition(IDEAL_SLIPPAGE_BPS);
+        assertApproxEqRel(strategy.totalLockedValue(), assets, 0.01e18);
 
-            strategy.endPosition(IDEAL_SLIPPAGE_BPS);
-            assertApproxEqRel(strategy.totalLockedValue(), assets, 0.01e18);
-        }
+        strategy.endPosition(IDEAL_SLIPPAGE_BPS);
+        assertApproxEqRel(strategy.totalLockedValue(), assets, 0.01e18);
+    }
+
+    /// @notice testing with multiple strategy with custom value
+    /// @dev be careful about the collateral ratio, must be lower than liquidation ratio
+    function testCustomLendingParamsFuzz(uint256 collateralToBorrowRatioBps) public {
+        // testing in range of 30% to 60%
+        collateralToBorrowRatioBps = (collateralToBorrowRatioBps % 3000) + 3000;
+        uint256 assetToDepositRatioBps = MAX_BPS.mulDivDown(MAX_BPS, MAX_BPS + collateralToBorrowRatioBps);
+
+        DeltaNeutralLp strategy2 = _deployStrategyWithCustomParams(
+            LendingParams({
+                assetToDepositRatioBps: assetToDepositRatioBps,
+                collateralToBorrowRatioBps: collateralToBorrowRatioBps
+            })
+        );
+
+        // add strategy to vault
+        vm.startPrank(governance);
+        vault.addStrategy(strategy2, 5000);
+
+        deal(address(asset), address(strategy2), startAssets);
+
+        assertTrue(strategy2.canStartNewPos());
```

Comment:
This assert isn't very useful. The lending params have nothing to do with whether or not the strategy can start a position.